lib/puppet/defaults.rb

```diff
@@ -305,7 +305,9 @@ def self.default_vendormoduledir
     :manage_internal_file_permissions => {
         :default  => ! Puppet::Util::Platform.windows?,
         :type     => :boolean,
-        :desc     => "Whether Puppet should manage the owner, group, and mode of files it uses internally",
+        :desc     => "Whether Puppet should manage the owner, group, and mode of files it uses internally.
+        
+          **Note**: For Windows agents, the `Default` is `false` for versions 4.10.13 and greater, versions 5.5.6 and greater, and versions 6.0 and greater.",
```

Comment:
Thanks @hestonhoffman. One nit, I'd leave it as "the default is `false` ...", as `Default` isn't a variable or something users can assign to/change.

Reply:
Thanks @joshcooper, I'll change that.